admin-dev/themes/default/template/controllers/search/helpers/view/view.tpl

```diff
@@ -58,7 +58,6 @@ $(function() {
                 {foreach $feature key=k item=val name=feature_list}
                     <tr>
                         <td><a href="{$val.link}"{if $smarty.foreach.feature_list.first}><strong>{$key}</strong>{/if}</a></td>
-                        <td><a href="{$val.link}">{$val.value}</a></td>
```

Comment:
Hello, this seems not related to your PR. Is it something you improved because you found it while preparing this Pull Request?

Reply:
On the contrary, this is very much related. This matches the empty right column in the "caractéristiques" list that you can see on the screenshot above. I removed it because, as you can see, there is no content to put there anymore.